.travis.yml

```diff
@@ -98,6 +98,8 @@ install:
 script: |
   set -e
   if [[ $JOB = docs ]]; then
+    # install docs dependencies
+    pip install '.[docs]'
```

Comment:
I don't really understand you last question here. Are you just confirming and rewording what I tried to say or is there some opinion of mine that you query?

The new changes you made seem good.